### PR TITLE
Stuff 2017

### LINKED ISSUE #873: Stuff 2017
- [x] Strangers
- [x] Wrecked
- [x] The Weird and the Eerie
- [x] Improvabunga
- [x] Debris
- [x] An Evening of Performance Art
- [x] Speechless
- [x] Escape for Dummies
- [x] Twenty-Something
- [x] Footnotes
- [x] The Fall of Byron Montrose: Poet, Gentleman, Lover
- [x] The Iconoclasts
- [x] Stuff Your Face With Poetry

### PR DESCRIPTION
**Pull request by samozzy**
All the StuFF 2017 shows, to close #873

### CODE REVIEW

File: _shows/16_17/wrecked_stuff.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ - role: 
+   name: Emma White
+ - role: 
+   name: Callum Walker
+ - role: 
+   name: Felicity Chilver
+ - role: 
+   name: Andrew Houghton
+ - role: 
+   name: Miguel Barrulas
```
Comment (BenWoodford76) [inline _shows/16_17/wrecked_stuff.md]: can we have an empty role field?

Comment (nhyty) [inline _shows/16_17/wrecked_stuff.md]: Yes

File: _shows/16_17/the_iconoclasts.md [hunk 2]
```diff
@@ -14,8 +14,7 @@ assets:
 crew:
   - role: Venue Technician
     name: Nathan Penney
-  - name: Joseph Heil
```
Comment (BenWoodford76) [inline _shows/16_17/the_iconoclasts.md]: I did help, didn't I??

Comment (nhyty) [inline _shows/16_17/the_iconoclasts.md]: i thought you went to get fish and chips, but u did do a little bit

Comment (BenWoodford76) [inline _shows/16_17/the_iconoclasts.md]: I also dealt with their LX guy and his strange demands ;) :)

**Comment by samozzy** [comment]
This is now done.

**Pull request merged**